app/mailers/transaction_mailer.rb

```diff
@@ -30,7 +30,7 @@ def transaction_preauthorized(transaction)
       payment_type = MarketplaceService::Community::Query.payment_type(@community.id)
       gateway_expires = MarketplaceService::Transaction::Entity.authorization_expiration_period(payment_type)
 
-      expires = Maybe(transaction).booking.end_on.map { |booking_end|
+      expires = Maybe(transaction).booking.end_on_exclusive.map { |booking_end|
```

Comment:
That's a tricky question. This will definitely change the behaviour: For day pickers, the expires date is now +1 day, if the `(booking end + 2 days)` < `gateway expiration`. It's quite tricky logic...

I'm not sure if I can say that it has been "broken" before. Why did you conclude that exclusive end_on is assumed here? It feels that this is not exact science due to the +2 days that is added by the `preauth_expires_at` method.